recipes/pyrost/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "pyrost" %}
+{% set version = "0.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3d97c37623c038a3ee63c74b17bce331e80f72156c81d40f55a7e2d860848668
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - {{ mpi }}
+    - openblas
+    - python
+    - pip
+    - cython
+    - gsl
+    - h5py
+    - numpy
+    - scipy
+  run:
+    - {{ mpi }}
+    - libopenblas
+    - openblas
+    - python
+    - gsl
+    - h5py
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - pyrost
+  requires:
+    - pytest
+  source_files:
+    - tests/test_pyrost.py
+    - pytest.ini
+  commands:
+    - pytest tests -m standalone
```

```suggestion
    - pytest tests -m standalone
    - pip check
```
Use pip check to confirm the dependencies agree.

---

Done, thanks for the suggestion.